charts/harmony-chart/templates/NOTES.txt

```diff
@@ -10,9 +10,13 @@ that the load balancer is working. First, get its external IP using
 Next, go to http://the.external.ip.shown/cluster-echo-test and make sure you get
 a JSON response.
 
+{{- if .Values.openfaas.enabled }}
+OpenFAAS is enabled. The default admin user password is automatically generated
```

Comment:
Could this statement link out to documentation showing some examples of how OpenFaas could be used or providing details on how to add a function?

Reply:
Although this could, I'm not convinced that the Harmony chart should contain "tutorial"/educational content. I mean, if someone enables OpenFAAS, probably the reason is that he/she knows why it is going to be enabled.